src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-server.port=${PORT:8080}
+server.port=${PORT:8090}
```

Comment:
NIT: You can leave that at `8080` and set the port manually as an environment variable.